citizen_science/citizen_science.md

```diff
@@ -0,0 +1,264 @@
+- text coding, such as in the creation of training data for machine learning algorithms ([Injury Coding](https://github.com/NASA-Tournament-Lab/CDC-NLP-Occ-Injury-Coding))
+- image analysis ([Placenta Profiles](https://www.zooniverse.org/projects/msbrhonclif/science-scribbler-placenta-profiles/about/research))
+- problem solving ([Hewman game](https://citizensciencegames.com/games/hewmen/))
+- writing and debugging code, including complex data science projects ([EvidentlyAI's Hacktoberfest](https://www.evidentlyai.com/hacktoberfest-2022))
+- donate spare CPU time for computationally intense analyses ([FoldingatHome](https://foldingathome.org/))
+
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Read [an interview with biologist Caren Cooper](https://e360.yale.edu/features/interview_caren_cooper_how_rise_of_citizen_science_is_democratizing_research) about her experience with several citizen science projects, including what she's learned about how participating in citizen science can impact individuals and communities.
+
+</div>
+
+Citizen science is attractive to many researchers not only because of the possibility of outsourcing work on large complex data, but because it provides the potential to democratize science and allow the public more influence over and involvement in the scientific process.
+
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Some projects focus especially on how public involvement in science can put power back in the hands of communities.
+For example, Public Lab works to pursue environmental justice through community science and open technology. From the [Public Lab About page](https://publiclab.org/about):
+
+"When people can easily and reliably track local effects associated with environmental injustices — increased flooding, poor air quality, pollution and destruction of wetlands — they can make better-informed decisions and take action."
+
+</div>
+
+
+The prevalence of citizen science has increased dramatically in recent years, especially as more digital platforms emerge to support it (for example [Zooniverse](https://www.zooniverse.org/), [Globe at Night](https://www.globeatnight.org/), [Foldit](https://fold.it/), and [The Many Brains Project](https://www.testmybrain.org/)).
+
+
+<div class = "history">
+<b style="color: rgb(var(--color-highlight));">Historical context</b><br>
+
+[What Is Citizen Science? – A Scientometric Meta-Analysis](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147152) explores the history of citizen science projects, including which fields make the most use of citizen science and what scientific output they have produced.
+It includes a meta-analysis of more than 2000 published papers.
+
+</div>
+
+## Why participate in citizen science?
```

Comment:
My initial thought was that this section would be about why a scientist would want to structure something as "citizen science" but I'm not sure how to rephrase this to make it clear that this section is about the citizens volunteering.

Reply:
Ah, good call. I changed it to "What motivates citizen scientists?"